src/data/tagsDataWithDetails.js

```diff
@@ -0,0 +1,178 @@
+<ruby>, 'Defines a ruby annotation (for East Asian typography)', FALSE, '', FALSE,
+<s>, 'Defines text that is no longer correct', FALSE, '', TRUE,
+<samp>, 'Defines sample output from a computer program', FALSE, '', FALSE,
+<script>, 'Defines a client-side script', FALSE, '', FALSE,
+<search>, 'Defines a search section', FALSE, '', TRUE,
+<section>, 'Defines a section in a document', FALSE, '', TRUE,
+<select>, 'Defines a drop-down list', FALSE, '', TRUE,
+<slot>, 'Defines a placeholder inside a web component that you can fill with your own markup', FALSE, '', FALSE,
+<small>, 'Defines smaller text', FALSE, '', FALSE,
+<source>, 'Defines multiple media resources for media elements (<video> and <audio>)', FALSE, '', FALSE,
+<span>, 'Defines a section in a document', FALSE, 'It\'s a generic inline-level container that has no special meaning. It should only be used when no other semantic element is relevant', FALSE,
+<strike>, 'Defines strikethrough text', TRUE, 'Use <del> or <s> instead', FALSE,
+<strong>, 'Defines important text', FALSE, '', TRUE,
+<style>, 'Defines style information for a document', FALSE, '', FALSE,
+<sub>, 'Defines subscripted text', FALSE, '', TRUE,
+<summary>, 'Defines a visible heading for a <details> element', FALSE, 'Many, but not all, user agents expose the summary element with an implicit role of button.', FALSE,
+<sup>, 'Defines superscripted text', FALSE, '', TRUE,
+<svg>, 'Defines a container for SVG graphics', FALSE, '', TRUE,
+<table>, 'Defines a table', FALSE, '', TRUE,
+<tbody>, 'Groups the body content in a table', FALSE, '', TRUE,
+<td>, 'Defines a cell in a table', FALSE, '', TRUE,
+<template>, 'Defines a container for content that should be hidden when the page loads', FALSE, '', FALSE,
+<textarea>, 'Defines a multiline input control (text area)', FALSE, '', TRUE,
+<tfoot>, 'Groups the footer content in a table', FALSE, '', TRUE,
+<th>, 'Defines a header cell in a table', FALSE, '', TRUE,
+<thead>, 'Groups the header content in a table', FALSE, '', TRUE,
+<time>, 'Defines a specific time (or datetime)', FALSE, '', TRUE,
+<title>, 'Defines a title for the document', FALSE, '', FALSE,
+<tr>, 'Defines a row in a table', FALSE, '', TRUE,
+<track>, 'Defines text tracks for media elements (<video> and <audio>)', FALSE, '', FALSE,
+<tt>, 'Defines teletype text', TRUE, 'Use CSS instead', FALSE,
+<u>, 'Defines some text that is unarticulated and styled differently from normal text', FALSE, '', FALSE,
+<ul>, 'Defines an unordered list', FALSE, '', TRUE,
+<var>, 'Defines a variable', FALSE, '', FALSE,
+<video>, 'Defines embedded video content', FALSE, '', FALSE,
+<wbr>, 'Defines a possible line-break', FALSE, '', FALSE,
+<xmp>, 'Renders text between the start and end tags without interpreting the HTML in between and using a monospaced font', TRUE, 'Use <pre> and <code> instead', FALSE,
+`
+
+// FOR REVIEWER TO SEE THE OUTPUT OF THE PARSED DATA: 
```

Comment:
Is this comment still valid? Because I cannot see an "export" in line 2 which can be uncomment.

Reply:
Oh, my bad. I meant to say "comment out" the "export"!

Reply:
I removed the export and I removed the comment from the code below. What exactly do you mean by `run the code`? Is `open with live server` enough? When doing so, I get an `Uncaught SyntaxError: ambiguous indirect export: tagsDataWithDetails`

Reply:
Hmm, interesting ... Not sure why that is.
Trying to be fancy now (without the right tools though. I have a very old laptop), so I [recorded the steps of how I run the code](https://watch.screencastify.com/v/A8xkpMeaHIy0Z110yjkI).
Also, made [a shorter version](https://watch.screencastify.com/v/9gyaOMf9OAITnALty3bm).

Hope at least one of the links works for you. We could add short videos to our arsenal of tools, it might speed up communication sometimes.

Reply:
Thanks for sharing this video, amazing work! 👏🏽 

Unfortunately, my tab bar looks quite differently like yours, and execute "run" from the main bar (with and without debugging) is doing weird things.